docs/sql/statements/start-transaction.rst

```diff
@@ -0,0 +1,42 @@
+.. highlight:: psql
+.. _ref-start:
+
+=====================
+``START TRANSACTION``
+=====================
+
+Start a transaction block
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+Synopsis
+========
+
+::
+
+   START TRANSACTION [ transaction_mode [ , ...] ]
+
+where ``transaction_mode`` is one of::
+
+   ISOLATION LEVEL isolation_level | (READ WRITE | READ ONLY) | [NOT] DEFERRABLE
+
+where ``isolation_level`` is one of::
+
+   { SERIALIZABLE | REPEATABLE READ | READ COMMITTED | READ UNCOMMITTED }
+
+Description
+===========
+
+The statement starts a transaction block until it committed or rolled back.
+
+As CrateDB does not support transactions, this command has no effect and will
+be ignored.
```

Comment:
```suggestion
CrateDB does not support transactions and will silently ignore the ``START
TRANSACTION`` statement.
```

something like this maybe? I'm cautious about documenting the _intended_ function of the statement when it is a no-op for CrateDB users. my guess is that it would be more confusing than it is useful. but again, happy to take input on this as I probably do not have the full picture